src/mon/NVMeofGwMon.cc

```diff
@@ -513,6 +592,18 @@ bool NVMeofGwMon::preprocess_beacon(MonOpRequestRef op)
   return false;
 }
 
+epoch_t NVMeofGwMon::get_ack_map_epoch(bool gw_created,
+    const NvmeGroupKey& group_key) {
```

Comment:
(not a request to change this PR, as it's urgent - and as Sam did OK it; but:)
early return is almost always easier to read than a chain of 'if else' with logic.
I.e. - this could have been:
```
epoch_t ..... {
if  (!gw_created) {
  return 0;
}
const auto group_epoch = map.gw_epoch.find(group_key) ;
if  (group_epoch == map.gw_epoch.end()) {
  // feature...
  return map.epoch;
}
return *group_epoch;
}

```
or similar